test/support/rails_application_fixture_helper.rb

```diff
@@ -0,0 +1,75 @@
+# typed: ignore
+# frozen_string_literal: true
+
+require "rails_test_helper"
+
+module RailsApplicationFixtureHelper
+  include ApplicationFixtureHelper
+
+  def setup_application_fixture
+    super()
+    cache_rails_paths
+  end
+
+  def teardown_application_fixture
+    restore_rails_paths
+    super()
+  end
+
+  def copy_template(template)
```

Comment:
did you mean `use_template`?

Reply:
Why yes, I did! The tests happen to pass because rails_test_helper.rb configures the Dummy application with Rails paths pointing to test/fixtures/skeleton/**, which happens to contain the same files as the copy directory.  And virtually all of the tests in ApplicationValidatorTest refer to skeleton, not minimal.

Good catch!